### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Auster-South-Anemoi**
Expanded the instructions and reworded some parts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
-This is a small windows script to help extract Wings! amiga disks from the GOG release.
+This is a small Windows script to help extract Wings! amiga disks and the Kickstart 1.2 firmware from the GOG release. Also possible to extract from Linux, as explained below.
```
Comment (sigboe) [inline README.md]: `extract from Linux`

Please change the grammar here, to something other than from Linux. Maybe using linux or running linux, or reword the whole sentence! :+1:

File: README.md [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
-7z should be in your path, I got this on Windows by installing 7zip
+To have [7-Zip](https://www.7-zip.org/) installed.
 
 # Instructions
 
-1. Install Wings!
-2. find the Wings.exe, typically in "C:\GOG Games\Wings! Classic"
-3. drag and drop it over the extractwings.bat file
-4. you should now have two .adf files wings-1.adf and wings-2.adf these are the game disks
+1. Open the command prompt and run the command `set PATH=%PATH%;C:\Program Files\7-Zip\` (change `C:\Program Files\7-Zip\` to where you installed 7-Zip, if you didn't install it in the default folder)
```
Comment (sigboe) [inline README.md]: For me installing 7zip added 7z to the path in Windows. Did this not happen for you?

File: README.md [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
+2. Download [extractwings's latest build](https://github.com/sigboe/extractwings/archive/refs/heads/master.zip)
```
Comment (sigboe) [inline README.md:10]: I like these instruction changes, but do you think we should put it as a release instead?

File: README.md [hunk 4]
```diff
@@ -1,21 +1,25 @@
...
+3. Unzip extractwings somewhere you can find
+4. Install Wings!
+5. Find the Wings.exe, typically in "C:\GOG Games\Wings! Classic"
+6. Drag and drop it over the extractwings.bat file
+7. You should now have three new files, wings-1.adf, wings-2.adf and Kick12.rom. The .adf files are your ROM, and Kick12.rom is the firmware file.
 
 # Linux
-For Linux, things are simpler
+For Linux, things are simpler. Just run these commands from the folder with Wings.exe using the terminal:
```
Comment (sigboe) [inline README.md]: I prefer not suggesting the directory to run the commands from.

**Review by sigboe — CHANGES_REQUESTED** [review]
What a great PR I have some requests before merging! :smile:

File: README.md (hunk 2 above)
Comment (Auster-South-Anemoi) [reply README.md]: At least not in the virtual machine I used to test. Maybe I should add a note about the path being possibly set already?

File: README.md (hunk 3 above)
Comment (Auster-South-Anemoi) [reply README.md:10]: A release sounds interesting too.

File: README.md (hunk 4 above)
Comment (Auster-South-Anemoi) [reply README.md]: I see. I'll remove it, then. Hopefully people know the commands need to be run from where the Wings.exe file is, or to point the command to the right folder.
Also, I just remembered these commands could possibly be compressed into a .sh script on Linux. I'll be testing it after doing the changes in the PR.

File: README.md (hunk 1 above)
Comment (Auster-South-Anemoi) [reply README.md]: I'll see how I can reword it. Thanks for pointing out.

File: README.md (hunk 4 above)
Comment (Auster-South-Anemoi) [reply README.md]: About the .sh script, it's surprisingly easy.
Just create a .sh file (or a .txt file and rename the extension .sh) and paste the commands in the README.md, then run this command with super user privileges ("sudo" argument in Ubuntu, not sure on other distros):
```chmod +x /path/to/script.sh```
After that, put the .sh file in the same folder as Wings.exe and either open the file directly if your OS allows, or run in the terminal (`cd /path/to/folder/with/the/files/` then `./script.sh`). The files should be extracted in the same folder.

Comment (Auster-South-Anemoi) [reply README.md]: Maybe you could add such .sh script to the downloadable files as well? (I'm not sure if/how I can add files through PR, just edit existing ones)

**Comment by Auster-South-Anemoi** [comment]
Added a new commit with the changes requested:
https://github.com/sigboe/extractwings/pull/3/commits/3764666c7ece853be7a633474a818b940d57f880

**Comment by Auster-South-Anemoi** [comment]
Just need to see if it's more interesting to add a .sh file or keep the Linux commands.
The upside of this is that it can run all commands from a single file/command, instead of manually using all commands.
Or maybe you need two commands, if you can't upload Linux scripts already set as executables, but I never tested to see if there's such a limitation. Still, 1-2 commands instead of 4.

**Pull request merged**